ext/curl/tests/curl_postfields_array.phpt

```diff
@@ -0,0 +1,63 @@
+--TEST--
+CURLOPT_POSTFIELDS with multi-value fields
+--EXTENSIONS--
+sockets
+--FILE--
+<?php
+$socket = stream_socket_server("tcp://0.0.0.0:29999", $errno, $errstr);
+
+if (!$socket) {
+	echo "$errstr ($errno)<br />\n";
+	return;
+}
+
+$url = "http://127.0.0.1:29999/get.inc?test=raw";
+
+$fields = [
+	'single' => 'SingleValue',
+	'multi' => [
+		'Multi1',
+		'Multi2',
+	]
+];
+
+$options = array(
```

Comment:
Any reason not to use the _short array syntax_ `[]` here?

Reply:
Not really, I probably copied it from the original example. Will fix.